### DISCUSSION TITLE
3D Object turn black when interact with it

### DISCUSSION THREAD

**Discussion by cuicmex** [google/model-viewer#3759] [Q&A]
In a Samsung S22 Ultra with Android and Chrome, sometimes when you start manipulating the 3D object it starts to turn black. It is not immediately, but a little with each manipulation. It also happens sometimes when moving the device back and forth (away and toward the 3D model)

If exits from AR experience and start again, then works fine again.

Chrome, version: 104.0.5112.97
Android, version: 12; SM-S908B Build/SP1A.210812.016

    <model-viewer bounds="tight" enable-pan src="painting.glb" ar ar-placement="wall"  ar-modes="scene-viewer quick-look webxr" camera-controls environment-image="neutral" poster="poster.webp" shadow-intensity="0" ar-scale="fixed">
      <div class="progress-bar hide" slot="progress-bar">
          <div class="update-bar"></div>
      </div>
      <button slot="ar-button" id="ar-button">
          View in your space
      </button>
      <div id="ar-prompt">
          <img src="ar_hand_prompt.png">
      </div>
    </model-viewer>  

https://user-images.githubusercontent.com/74239483/187670322-bc1b9bc8-f2d5-413c-a776-1a7309d2805f.mp4

**Comment by elalish** [comment]
This looks like a Scene Viewer bug, @eugenecys. WebXR mode should not have this problem; have you tried it?

**Reply by cuicmex** [reply]
I have tried with webxr, but if I set ar-modes="webxr scene-viewer quick-look", then the model loads, but the background with the camera view is not rendered, it just shows a black background.

**Reply by elalish** [reply]
Ah, this is a Samsung bug with this device. See #3495 and maybe #3364
